internal/policy/operator/deployable_by_olm.go

```diff
@@ -128,7 +130,7 @@ func (p *DeployableByOlmCheck) Validate(ctx context.Context, bundleRef certifica
 		return false, fmt.Errorf("%v", err)
 	}
 
-	log.L().Debugf("The operator Metadata is %+v", *operatorData)
```

Comment:
Do you know off hand if we'll get the same output from the "%+v" formatter using the k/v setup in the new logger?

Reply:
I do not. Will need to test that specifically.

Reply:
```
time="2022-12-02T08:18:25-05:00" level=debug msg="operator metadata" metadata="{\"CatalogImage\":\"quay.io/opdev/simple-demo-operator-catalog:v0.0.6\",\"Channel\":\"stable\",\"PackageName\":\"simple-demo-operator\",\"App\":\"simple-demo-operator\",\"InstallNamespace\":\"simple-demo-operator\",\"TargetNamespace\":\"simple-demo-operator-target\",\"InstallModes\":{\"AllNamespaces\":{\"type\":\"AllNamespaces\",\"supported\":true},\"MultiNamespace\":{\"type\":\"MultiNamespace\",\"supported\":false},\"OwnNamespace\":{\"type\":\"OwnNamespace\",\"supported\":true},\"SingleNamespace\":{\"type\":\"SingleNamespace\",\"supported\":false}},\"CsvNamespaces\":null,\"InstalledCsv\":\"\"}"
```